wdl2cwl/main.py

```diff
@@ -87,16 +87,17 @@ def get_command(
                             new_command += "$(inputs." + i + ")"
             # if sub string has only the input/ variable name
             else:
-                data_type = (
-                    input_types[input_names.index(sub_str)]
-                    if sub_str in input_names
-                    else ""
+                append_str = (
+                    '$(inputs["'
+                    + sub_str
+                    + '"]===null?"":inputs["'
+                    + sub_str
+                    + '"]["class"]==="File"? "$(inputs.'
+                    + sub_str
+                    + '.path)": "$(inputs.'
+                    + sub_str
+                    + ')")'
```

Comment:
Instead of using Javascript here, set the default value to `""` when you construct a cwl.CommandInputParameter for an optional input string that has now default value of its own

Reply:
Shall I keep '.path' for files or remove it?